### PR TITLE
Migrated codemirror-graphql to typescript

### PR DESCRIPTION
**Pull request by imolorhe**
Moving all codemirror-graphql logic to typescript.

### CODE REVIEW

File: packages/codemirror-graphql/package.json [hunk 1]
```diff
@@ -13,34 +13,28 @@
...
-    "lint.js.flow",
-    "mode.esm.js",
-    "mode.js",
-    "mode.js.flow",
+    "dist",
     "utils",
     "variables",
-    "results"
+    "results",
+    "/*.js",
```
Comment (acao) [inline packages/codemirror-graphql/package.json:23]: this strategy works but then we end up with `babel.config.js` and then `jest.config.js`. is it possible to mix include/ignore arguments now with files param?

Comment (imolorhe) [reply packages/codemirror-graphql/package.json:23]: Excluded babel and jest configs

**Review by acao — APPROVED** [review]
looks great! lets merge it

**Comment by acao** [comment]
oh crap @imolorhe we forgot the changeset ://

File: packages/codemirror-graphql/src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as CM from 'codemirror';
+import 'codemirror/addon/hint/show-hint';
+
+declare module 'codemirror' {
```
[the remark this answers is not in the document]
Comment (imolorhe) [reply packages/codemirror-graphql/src/index.d.ts:4]: We should be able to rename it without issues

**Comment by imolorhe** [comment]
Oh right! That can be another PR right?

**Pull request merged**